eagle-external/eagle-cdh/src/aux/eagle-server.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
Ok,I'll remove eagle-server.sh,it's very detailed code. And eagle.properties, please review https://github.com/apache/eagle/pull/908/files#diff-712f093104224c501339f2117db3b849R764 .